integration/test/Test/TeamCollaborators.hs

```diff
@@ -61,14 +64,35 @@ testCreateTeamCollaboratorPostTwice = do
   (owner, team, _members) <- createTeam OwnDomain 2
 
   -- At the time of writing, it wasn't clear if this should be a bot instead.
-  userId <- randomUser OwnDomain def >>= asString . (%. "id")
+  user <- randomUser OwnDomain def
   let add =
         addTeamCollaborator
           owner
           team
-          userId
+          user
           [ "create_team_conversation",
             "implicit_connection"
           ]
   bindResponse add assertSuccess
   bindResponse add $ assertStatus 409
+
+-- Question about channels: Do collaborators get to create them when all team members are allowed to create them?
```

Comment:
The test is about conversations, right?
So, have you forgotten to pose the question?

Reply:
```suggestion
```

Answered in chat, no channels for now.